src/pykeen/losses.py

```diff
@@ -953,6 +954,63 @@ def forward(
         return loss
 
 
+@parse_docdata
+class FocalLoss(PointwiseLoss):
+    """A module for the focal loss proposed by [lin2018]_.
+
+    It is an adaptation of the (binary) cross entropy loss, which deals better with imbalanced data.
+    The implementation is strongly inspired by the implementation in
+    :func:`torchvision.ops.sigmoid_focal_loss`, except it is using
+    a module rather than the functional form.
+
```

Comment:
@mberr can you add some fancy math here?

Reply:
done in e1dc21d